### DISCUSSION TITLE
Sync Schedule from Git Storage Module not working

### DISCUSSION THREAD

**Discussion by HeatZeeker** [requarks/wiki#3573] [Error / Bug Report]
Hi, I can't save a different schedule sync in the Git Storage Module.

 - OS: Debian 9
 - Wiki.js version: 2.0.0-rc.1
 - Database engine: postgres 9.6

**Comment by maxproske** [comment]
I am also experiencing the same visual bug. It successfully updates the `syncInterval` to `PT1H` in the database, but the admin console always displays "5 minutes". Here is a screenshot of the console:

![Error](https://i.imgur.com/drzu3Dq.png)

**Host Info**
- OS: Ubuntu 18.04, Docker
- Wiki.js version: 2.0.0-rc.17
- Database engine: postgres 11.5
